openmc/universe.py

```diff
@@ -288,8 +309,17 @@ def get_all_cells(self):
 
         return cells
 
-
     def get_all_universes(self):
+        """Return all universes that are contained within this one if it is filled with
```

Comment:
might clarify this as "within this one if any of its cells are filled with a universe or lattice"